src/main/flight/imu.c

```diff
@@ -87,14 +86,18 @@ static float throttleAngleScale;
 static float fc_acc;
 static float smallAngleCosZ = 0;
 
-static float magneticDeclination = 0.0f;       // calculated at startup from config
```

Comment:
I do not think so .
It is not used in betaflight code i checked it,
but perhaps it's very well hidden, so i didn't find it.
You could give me a hint ... where it is used in code.